eng/docker/alpine.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.1.0-preview1-runtime-deps-alpine
+FROM mcr.microsoft.com/dotnet/runtime-deps:2.1-alpine
```

Comment:
Note this was already fixed in release/2.1 and we don't use bionic Docker images in that branch